glib/src/convert.rs

```diff
@@ -317,6 +317,65 @@ pub fn locale_to_utf8(opsysstring: &[u8]) -> Result<(crate::GString, usize), Cvt
     }
 }
 
+#[doc(alias = "g_utf8_to_ucs4")]
+#[doc(alias = "g_utf8_to_ucs4_fast")]
+pub fn utf8_to_utf32(str: &str) -> Slice<char> {
```

Comment:
all the new manually written functions should be added to the Gir.toml and marked as manual, otherwise they won't be automatically documented

Reply:
also if the function was renamed like in this case, it has to be configured in the Gir.toml as well for the docs links to work properly if there are any

Reply:
At least for the `utf8_` functions this isn't trivial, as there's a section in the `Gir.toml` ignoring all of them, and adding specific sections for individual ones didn't seem to generate docs for them in my attempts, but maybe I did it wrong?

For the `ucs4_to_utf8` function, we could just replace the current regex that ignores everything starting with `ucs4_` to one only ignoring `ucs4_to_ucs16`, but I still couldn't get it to generate docs just for `utf32_to_utf8` (renamed from `ucs4_to_utf8`) even with that.

Any tips?

Reply:
If you can push your changes, I can try and see what is going wrong afterwards.

Reply:
I pushed my changes in 456cecdb6a34ae0947cf88cbde01af8c0a882f92

Reply:
@bilelmoussaoui What's the status here?

Reply:
I wouldn't block the PR on that, I will try to find some time during the end of year holidays and improve the docs situation for glib/gio & other gtk-rs-core libraries.